com.ibm.jbatch.tck.ann.proc/pom.xml

```diff
@@ -17,33 +17,34 @@
 
    SPDX-License-Identifier: Apache-2.0
  -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>jakarta.batch</groupId>
-    <artifactId>tck-parent</artifactId>
-    <version>1.0.2</version>
-  </parent>
-  <artifactId>com.ibm.jbatch.tck.ann.proc</artifactId>
-  <name>Jakarta Batch TCK Annotation Processor</name>
-  
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <configuration>
-          <compilerArgument>-proc:none</compilerArgument>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
-  
-  <dependencies>
-    <dependency>
-      <groupId>jakarta.batch</groupId>
-      <artifactId>com.ibm.jbatch.tck.ann</artifactId>
-    </dependency>
-  </dependencies>
-  
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Maybe before we're all done we can go back and update the copyright in all the files (or it could be another PR).  Something like:

> Copyright 2012, 2020 International Business Machines Corp. and others

Reply:
Regarding the copyright I'd open an issue for backtracking it and do the change with a different PR.